src/test/java/org/example/MainWindowTest.java

```diff
@@ -0,0 +1,40 @@
+package org.example;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class MainWindowTest {
+
+    MainWindow mainWindowTest;
+    MyTable table;
+
+    @BeforeEach
+    void setUp() {
+        MyTable.setInstance(table);
+        table = mock(MyTable.class);
+        mainWindowTest = new MainWindow();
+    }
```

Comment:
I think u need to use headless system, as I see the reason of error in CI

Reply:
Idk but maybe it will help.. :/

Reply:
> I think u need to use headless system, as I see the reason of error in CI



> I think u need to use headless system, as I see the reason of error in CI



> I think u need to use headless system, as I see the reason of error in CI

nothing changed.....